### PR TITLE
Progress indicator for attachment uploads

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob Röhrl <jakob.roehrl@web.de>
* Resolves: #1597 
* Target version: master

### CODE REVIEW

**Comment by jakobroehrl** [comment]
@juliushaertl 
This doesn't work so far. 
Do you see an error?

File: src/services/AttachmentApi.js [hunk 1]
```diff
@@ -38,10 +38,18 @@ export class AttachmentApi {
...
+			  const percentCompleted = Math.round((progressEvent.loaded * 100) / progressEvent.total)
+			  console.log(percentCompleted)
+			},
+		}
+
 		const response = await axios({
 			method: 'POST',
 			url: this.url(`/cards/${cardId}/attachment`),
 			data: formData,
+			config: config,
```
Comment (juliushaertl) [inline src/services/AttachmentApi.js:52]:
```suggestion
			...config,
```

As the first parameter to axios() is the config already.

Comment (jakobroehrl) [reply src/services/AttachmentApi.js:52]: Thanks, now I get a console log result. 
How can I access this value from the view components?

Comment (juliushaertl) [reply src/services/AttachmentApi.js:52]: Well the easiest would be to pass a callback to the service method and then being able to access it directly in the component.

**Comment by jakobroehrl** [comment]
@juliushaertl 
Could you check this please, thanks.

**Comment by jakobroehrl** [comment]
@juliushaertl 
Looks good, let's merge this in

**Pull request merged**